docs/arlas-server-configuration.md

```diff
@@ -60,6 +60,14 @@ docker run -ti -d \
 
 ## ARLAS configuration properties
 
+### Authentication
+
+| Environment variable | ARLAS Server configuration variable | Default | Description |
+| --- | --- | --- | --- |
+| ARLAS_AUTH_ENABLED | arlas_auth.enabled | false | Wether to enable auth or not |
```

Comment:
Whether

Reply:
Should be fixed by https://github.com/gisaia/ARLAS-server/compare/f020bff4d4a5b1649e12b8ab457dcb79dc54615a..3b14f507b53c59ae358dcbc33601b9e8eb2138f9